content/concepts/dolthub/api.md

```diff
@@ -234,3 +236,66 @@ res.json()
 ```
 
 Please note: You must include a ref name (branch, tag, commit hash, etc) when making authenticated calls to the SQL API using a token. Unauthenticated API requests do not require this. They use the default branch (`main` or `master`).
+
+## CSV API
+
+CSV API allow users to fetch CSV files of data from databases. We provide CSV files of database tables, individual files as well as a ZIP file of the database at a commit or branch.
+
+### Example
+
+#### Reading
+
+We will use an example DoltHub database, [dolthub/us-jails](https://www.dolthub.com/repositories/dolthub/us-jails/) and the Python `requests` library to explore it in the Python console.
+
+##### Table
+
+Download the table `incidents` from `main` branch:
+
+```python
+import requests
+local_file = 'incidents_main.csv'
+res = requests.get('https://www.dolthub.com/csv/dolthub/us-jails/main/incidents')
+with open(local_file, 'wb')as file: file.write(res.content)
```

Comment:
I feel like this is easier to read for all of these if you have a line break (and space before `as`)
```suggestion
with open(local_file, 'wb') as file: 
    file.write(res.content)
```